conf/app.routes

```diff
@@ -7,4 +7,5 @@ GET        /assets/*file                                            @controllers
 GET        /language/:language                                      @langswitch.LanguageSwitchController.switchToLanguage(language: Language)
 POST       /manage-or-track-submit                                  controllers.Controller.manageOrTrackSubmit
 GET        /signout                                                 controllers.Controller.signout
-GET        /bank-account-coc/start-journey/vrn/:vrn/:returnPage     controllers.Controller.startBankAccountCocJourney(vrn : Vrn, returnPage : ReturnPage)
\ No newline at end of file
+GET        /bank-account-coc/start-journey/vrn/:vrn/:returnPage     controllers.Controller.startBankAccountCocJourney(vrn : Vrn, returnPage : ReturnPage)
+GET        /view-progress/vrn/:vrn/:periodKey                       controllers.Controller.viewProgress(vrn : Vrn, periodKey : PeriodKey)
```

Comment:
I'm still not sure if it's safe to expose VRN in url. It can be changed  by hand.